src/docs/output-targets/custom-elements.md

```diff
@@ -68,6 +68,15 @@ This flag defaults to `false` when omitted from a Stencil configuration file.
 > Note: At this time, components created not using JSX may not be automatically
   defined. This is a known limitation of the API and users should be aware of
   it
+### externalRuntime
+
+When `true`, this flag results in the following behaviors:
+
+1. Minification will follow what is specified in the [Stencil config](docs/config#minifyJs).
+2. Filenames will not be hashed
+3. When resolving all `@stencil/core/*` imports as a part of the rollup process all of these resolved modules will be marked as external and therefore not included in the bundle.
```

Comment:
Open to input here on more context or a way to group all these behaviors into a good description for what this really does 🤷‍♂️ Behaviors just feel disconnected from each other

Reply:
Possibly it would be good to rephrase this along the same lines as the other entries in this list:

```suggestion
When `true`, this flag results in the following behaviors:

1. Minification will follow what is specified in the [Stencil config](docs/config#minifyJs).
2. Filenames will not be hashed
3. All imports from packages under `@stencil/core/*` will be marked as external and therefore not included in the bundle by Rollup.
```